app/src/main/java/org/kiwix/kiwixmobile/main/KiwixReaderFragment.kt

```diff
@@ -50,6 +51,8 @@ import org.kiwix.kiwixmobile.webserver.ZimHostActivity
 import org.kiwix.kiwixmobile.zim_manager.ZimManageActivity
 import java.io.File
 
+private const val HOME_URL = "file:///android_asset/home.html"
```

Comment:
this should be deleted and I think the home.html files should be deleted too

Reply:
Then I think this has a knockon effect of we get to delete all this weird `REPLACE_` handling, `ctrl+shift+f` `REPLACE_` to see what I mean